src/js/extension-anchor.js

```diff
@@ -24,6 +24,10 @@ var AnchorExtension;
 
         // Button and Extension handling
 
+        // labels for the anchor-edit form buttons
+        formSaveLabel: '&#10003;',
+        formCloseLabel: '&times;',
```

Comment:
I wonder if we should make the `fa` labels be exposed here as well for consistency. Thoughts?